nilearn/interfaces/fmriprep/exceptions.py

```diff
@@ -0,0 +1,28 @@
+class MissingConfound(Exception):
+    """
+    Exception raised when failing to find params in the confounds.
+
+    Parameters
+    ----------
+    params : list of missing params, default=[]
+    keywords: list of missing keywords, default=[]
+    """
+
+    def __init__(self, params=None, keywords=None):
+        """Set missing parameters and keywords."""
+        self.params = params or []
+        self.keywords = keywords or []
+
+
+class AllVolumesRemoved(Exception):
+    """
+    Exception raised when all volumes have either motion outliers or are non-steady-state volumes (`sample_mask` is an empty array).
+    """
+
+    def __init__(self):
+        super().__init__(
+            "All volumes are either outliers or non-steady-state. The size of the sample mask is 0."
```

Comment:
I think I would rephrase this part of the message into something:
"All time points in the confounds were marked as outliers. This would lead to all volumes in the time series to be scrubbed."
Just in case the confounds people pass are not just about non steady state.

Reply:
I second Remi. The case of all volumes are marked as non steady state is extremely unlikely. 

```
All time points in the confounds were marked as outliers. This would lead to all volumes in the time series to be scrubbed. 
```